pkgs/development/tools/jpexs-ffdec/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchzip, makeWrapper, makeDesktopItem, jdk8 }:
 
 stdenv.mkDerivation rec {
-  pname = "jpexs";
+  pname = "jpexs-ffdec";
```

Comment:
Why not ffdec?

and please add an alias that throws an error with the new package name.